ontologies/vocabs.ttl

```diff
@@ -340,8 +331,10 @@ Specifies a POSIX account"""^^xsd:string ;
 
 gist:_AccountType_windows-domain
 	a gist:AccountType ;
-	rdfs:label "Windows Domain"^^xsd:string ;
-	skos:altLabel "Windows-Domain"^^xsd:string ;
+	skos:altLabel
+		"Windows Domain"^^xsd:string ,
+		"Windows-Domain"^^xsd:string
```

Comment:
Is this helpful? 
Also, should there not be a stixTerm? I guess that is a different git issue?